fleetspeak/src/client/client.go

```diff
@@ -253,48 +251,32 @@ func (c *Client) Stop() {
 	}
 }
 
+type serviceBytes struct {
+	// Used for pretty logging only.
+	path  string
+
+	bytes []byte
+}
+
 func (c *Client) loadServices() {
 	if c.cfg.ConfigurationPath == "" {
 		return
 	}
-	p := path.Join(c.cfg.ConfigurationPath, "services")
-	i, err := os.Stat(c.cfg.ConfigurationPath)
-	if err != nil {
-		log.Warningf("Unable to stat services path [%s], not loading services: %v", p, err)
-		return
-	}
-	if !i.Mode().IsDir() {
-		log.Warningf("Services path [%s] is not a directory, not loading services.", p)
-		return
-	}
 
-	d, err := os.Open(p)
+	ss, err := readServices(c.cfg.ConfigurationPath)
 	if err != nil {
-		log.Warningf("Unable to open services path [%s], not loading services: %v", p, err)
+		log.Warningf("Not loading services: %v", err)
```

Comment:
Nit: suggest making it a little more clear that we tried and failed, e.g. "Unable to load services: %s".

Reply:
Done.